upgrade/helpers/tasks.py

```diff
@@ -799,3 +799,33 @@ def pre_upgrade_system_checks(capsules):
     if capsules:
         for capsule in capsules:
             wait_untill_capsule_sync(capsule)
+
+
+def generate_custom_certs():
```

Comment:
I think, This task is more aligned with https://github.com/SatelliteQE/automation-tools than https://github.com/SatelliteQE/satellite6-upgrade

Reply:
@jyejare I have a scenario where I need some tasks from automation tools and some upgrade related task from satellite6-upgrade. Satellite6-upgrade has requirement of automation tools but automation-tools don't have satellite6-upgrade dependency, so I can easily clone satellite6-upgrade and use required automation tools task to achieve it. 
https://github.com/SatelliteQE/satellite6-upgrade/blob/master/requirements.txt#L2